server/src/constants/errors.js

```diff
@@ -3,6 +3,9 @@ const errorMessages = Object.freeze({
   nonUniqueEmail: 'This email is already associated with an account',
   noMatchingEmail: 'This email is not associated with an account',
   incorrectPassword: 'This password is incorrect',
+  onlySingleDrillingWell: 'Only one well per rig can be drilling at a time',
```

Comment:
I really like that you are taking the time to make your error messages specific. Nothing I hate more than a generic error message